.github/workflows/cut-release-branch.yml

```diff
@@ -91,7 +91,6 @@ jobs:
       - name: "Checkout ${{ github.repository }}"
         uses: actions/checkout@v3
         with:
-          ref: "main"
```

Comment:
@emmyoop will we want to re-add the ref to main here?

Reply:
I don't believe so.